### ISSUE TITLE
Use Other Dataset

### ISSUE THREAD

**Issue by longbai1006** [Lornatang/DBPN-PyTorch#1]
Thank you so much for implementing these SR baselines based on up-to-date PyTorch versions!

I tried to reproduce your DBPN-PyTorch on some other dataset, and I follow the instruction in data/readme. However, when I conduct the Step 3 python run.py, the code will generate 2 folders: DBPN/train & DBPN/valid, while these 2 folders remain **empty**. I am not sure why this happened.

My Step 2's train set structure is as follows, I replaced the DIV2K images with my own dataset:
- DIV2K
    - original
        - DIV2K_train_HR
            - 0001.png
            - 0002.png
            ...
            - 4300.png
        - DIV2K_valid_HR
            - 4301.png
            - 4302.png
            ...
            - 5748.png

0001-4300 is train set, and 4301-5748 is validation set. All images are 256x256x3.

Thank you very much in advance!

**Comment by Lornatang** [comment]
You need to modify image size=200, step=100, lines 17 and 18 in run.py, and run again to get the result you expect

**Comment by bt206636231** [comment]
Got it. Thanks bro!

**Comment by longbai1006** [comment]
Got it. Thanks bro!
